### ISSUE TITLE
"Peer died" when feerate_per_kw too high during fundchannel

### ISSUE THREAD

**Issue by jb55** [ElementsProject/lightning#443]
Looks like error isn't getting reported here:

peer-a: 

```
$ cli/lightning-cli fundchannel 021ba1e2bba7fda4f396d56fe3fb1f90c6fd3163ba3a677874c31f6e40839a39f0 500000   
"Peer died"
```

peer-a logs:

```
lightningd(8013): peer 021ba1e2bba7fda4f396d56fe3fb1f90c6fd3163ba3a677874c31f6e40839a39f0: state: UNINITIALIZED -> OPENINGD
lightning_openingd(9247): pid 9247, msgfd 19
lightning_openingd(9247): TRACE: First per_commit_point = 02699afac8633e58da720a6cb4946e973f21181f571c5fded8e7349cc6dab56668
lightning_openingd(9247): TRACE: Read decrypt 0011fa192c0c5bac7dd0f48e1dcbf54e44b457b32e1c061adc96b64ca4a0951354a20028666565726174655f7065725f6b772031373736322061626f7665206d6178696d756d203133363500
lightning_openingd(9247): STATUS_FAIL_PEER_BAD: Parsing accept_channel 0011fa192c0c5bac7dd0f48e1dcbf54e44b457b32e1c061adc96b64ca4a0951354a20028666565726174655f7065725f6b772031373736322061626f7665206d6178696d756d203133363500
lightningd(8013): peer 021ba1e2bba7fda4f396d56fe3fb1f90c6fd3163ba3a677874c31f6e40839a39f0: Peer permanent failure in OPENINGD: Parsing accept_channel 0011fa192c0c5bac7dd0f48e1dcbf54e44b457b32e1c061adc96b64ca4a0951354a20028666565726174655f7065725f6b772031373736322061626f7665206d6178696d756d203133363500
lightningd(8013):jcon fd 15: Failing: Peer died
lightning_gossipd(8022): TRACE: Forgetting remote peer 021ba1e2bba7fda4f396d56fe3fb1f90c6fd3163ba3a677874c31f6e40839a39f0
```

peer-b logs:
```
lightningd(4511): peer 02058ea66978d2e5c346642c45f3cec5a4c733897b59244fc29c3107e844f62608: state: UNINITIALIZED -> OPENINGD
lightning_openingd(4978): pid 4978, msgfd 18
lightning_openingd(4978): TRACE: First per_commit_point = 02abea1968573db289a0ffedc8a41b75124fe7f318024553cc2f0bffd19f5a304c
lightning_openingd(4978): STATUS_FAIL_PEER_BAD: feerate_per_kw 17762 above maximum 1365
lightningd(4511): peer 02058ea66978d2e5c346642c45f3cec5a4c733897b59244fc29c3107e844f62608: Peer permanent failure in OPENINGD: feerate_per_kw 17762 above maximum 1365
lightning_openingd(4978): Status closed, but not exited. Killing
lightning_gossipd(4520): TRACE: Forgetting remote peer 02058ea66978d2e5c346642c45f3cec5a4c733897b59244fc29c3107e844f62608
```

**Comment by 2pac1** [comment]
i was experiencing the same error following guide from https://interfect.github.io/#!/posts/009-Ride-the-Lightning.md

**Comment by rustyrussell** [comment]
This was fixed in commit 6b232de7b12755174c9d2b11d2c4318c14e15889 ("openingd: return to master for more gossip when negotiation fails").
